manifests/profile/apache.pp

```diff
@@ -69,13 +69,15 @@
     ensure  => 'directory',
     recurse => true,
     force   => true,
-    purge   => true
+    purge   => true,
+    require => Class['apache'],
   }
 
   file { '/etc/apache2/conf-available':
-    ensure => 'absent',
-    force  => true,
-    purge  => true
+    ensure  => 'absent',
+    force   => true,
+    purge   => true,
+    require => Class['apache'],
```

Comment:
This confuses me, but it doesn't look like it could cause any harm, so it's fine. I just don't see why requiring a directory be absent would need to require the apache class.

Reply:
Yeah, that one did not throw an error. Just put it in for parity.